### DISCUSSION TITLE
有没有完整的文档来说明v4和以前版本的区别

### DISCUSSION THREAD

**Discussion by hikaruwin** [httprunner/httprunner#1381] [2 - 答疑（Q&A）]
有没有完整的文档来说明v4和以前版本的区别，我使用下载的v4.1.4版本hrp命令建的脚手架已经没有api这个文件夹了，另外pip安装的v4.1.4和下载的v4.1.4有什么区别，都没有文档来说明

**Comment by debugtalk** [comment]
@hikaruwin 可以看下这个
https://httprunner.com/blog/release-v4/
